Extensions/View+asImage.swift

```diff
@@ -15,7 +15,7 @@ extension View {
         let view = controller.view
         let targetSize = controller.view.intrinsicContentSize
         view?.bounds = CGRect(origin: CGPoint(x: 0, y: 0), size: targetSize)
-        view?.backgroundColor = .clear
+        //view?.backgroundColor = .clear
```

Comment:
Not sure it does much but may as well return this to how it was?

Reply:
you're right, left it out while testing